shadow_sampler.py

```diff
@@ -0,0 +1,104 @@
+import os.path as osp
+
+import argparse
+import torch
+import torch.nn.functional as F
+from torch_geometric.datasets import Flickr
+from torch_geometric.loader import GraphSAINTRandomWalkSampler
+from torch_geometric.loader import ShaDowKHopSampler
+from torch_geometric.nn import GraphConv
+from torch_geometric.utils import degree
+
+path = osp.join(osp.dirname(osp.realpath(__file__)), '..', 'data', 'Flickr')
+dataset = Flickr(path)
+data = dataset[0]
+row, col = data.edge_index
+data.edge_weight = 1. / degree(col, data.num_nodes)[col]  # Norm by in-degree.
```

Comment:
I don't think we need this normalization here, as `ShaDowKHopSampler` does not perform a re-weighting.

Reply:
Yes, you are right. I have removed that part.